winpr/include/winpr/stream.h

```diff
@@ -287,6 +287,15 @@ extern "C"
 		*_s->pointer++ = (_v)&0xFF;
 	}
 
+    static INLINE void Stream_Write_UINT24_BE(wStream* _s, UINT16 _v)
```

Comment:
sure this should be `UINT16` and not `UINT32`?

Reply:
good catch